### ISSUE TITLE
Gradient support for stroke [FEATURE REQUEST]

### ISSUE THREAD

**Issue by AwixStudio** [simonoliver/UnityFigmaBridge#36]
It looks like gradients only work for fill image. Is there any chance to use gradient for stroke? 

For example to achieve this kind of effect on rectangle outline:
![image](https://github.com/simonoliver/UnityFigmaBridge/assets/29355013/80c91db1-e817-4a1f-9354-1dd466d9489b)

**Comment by simonoliver** [comment]
Yeah it wouldn't be too hard to add this, building on the same system used for gradient fill on the center. I'm actually reducing my time on Unity focussed packages (See here) - https://github.com/simonoliver/UnityFigmaBridge/discussions/35 - but this would be a nice easy one to add if I do invest more time on the package.

**Comment by AwixStudio** [comment]
Oh that's really sad :(. I guess I have to figure it out on myself (really need this). I will create pull request if I succeed.

**Comment by simonoliver** [comment]
Happy to answer any questions you have if you want some pointers.
